website/blog/2023-04-14-release-0.14.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Release Notes - Podman Desktop 0.14
+description: Podman Desktop 0.14 has been released! 
+slug: podman-desktop-release-0.14
+authors: [deboer]
+tags: [podman-desktop, release, kubernetes, kind]
+hide_table_of_contents: false
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 0.14 - Our Kind-est release yet!
+
+We have been working on a Kind extension for a while now, and decided it is time to promote it
+into the release just in time for KubeCon and CloudNativeCon Europe!
+
+Some of these features were developed in the Kind extension over the last few releases,
+but since they are new to you, and they were not in previous release notes, we will do a
+roundup and talk about all the Kind features.
+
+<!--Main Features-->
+
+- **Kind Installation**: Install Kind from the status bar
+- **Kind Clusters in Resources**: Create and manage Kind clusters from Resources
+- **Sharing Images with Kind**: Easily move images to Kind
+- **Kind Ingress**: Install an ingress controller
+- **Play YAML on Kubernetes**: Deploy a YAML file to a cluster
+- **UX and UI Improvements**: Updated preferences and telemetry prompt
+
+Podman Desktop 0.14 is now available. [Click here to download it](/downloads)!
+
+![Podman-desktop-0-14-hero](img/podman-desktop-release-0.14/podman-desktop-release-0.14.png)
+
+<!--truncate-->
+_________________
+
+## Release details
+
+### Kind Installation
+
+The Kind extension is now bundled as part of Podman Desktop [1421](https://github.com/containers/podman-desktop/issues/1421)
```

Comment:
let's show off this.
Put screenshots and avoid the reader to have to jump into bunch of PRs.

Explain to the readers how they can benefit from the capabilities?
- What do I need to do to install Kind?
- What does it do, what are the permission needed? 
- Where is the binary for Kind installed
- Learn more on our documentation

Once I have `kind` in the CLI, maybe we want to point to kind documentation?